tuning/sensorless.md

```diff
@@ -0,0 +1,477 @@
+---
+layout: default
+title: Configuring sensorless homing
+parent: Tuning Guides
+nav_exclude: true
+---
+
+This writeup covers how to set up sensorless homing on an Octopus 1.1 with 2209 drivers. Different drivers and/or MCUs may differ. Adjust accordingly.
+
+Also note that there are example working configurations at the bottom of this guide. Use them as examples based on your MCU and drivers, etc.
+
+{: .note }
+IF YOU DO NOT READ THIS CAREFULLY AND ADJUST THINGS WHEN NEEDED YOU WILL BE SAD. IF YOU USE A PHYSICAL ENDSTOP YOU MUST CHANGE THINGS TO HOME Z TO THE ENDSTOP PIN, NOT THE MIDDLE OF THE BED. YOU HAVE BEEN WARNED!
```

Comment:
Not a blocker for merge but maybe making it clearer that the user will need to update macros, specifically, would be useful?
```suggestion
IF YOU DO NOT READ THIS CAREFULLY AND ADJUST THINGS WHEN NEEDED YOU WILL BE SAD. IF YOU USE A PHYSICAL ENDSTOP YOU MUST CHANGE MACROS TO HOME Z TO THE ENDSTOP PIN, NOT THE MIDDLE OF THE BED. YOU HAVE BEEN WARNED!
```

Reply:
i agree. its implied in there but this sets them up for success better